spid-testenv/config.yaml

```
@@ -20,12 +20,8 @@ cert_file: "conf/idp.crt"
 # deve coincidere con quello presente nei metadati).
 # cfr. https://pysaml2.readthedocs.io/en/latest/howto/config.html#metadata
 metadata:
-  local:
-    - "conf/sp_metadata.xml"
-#  remote:
-#  - url: "http://spid-sp/metadata
-#    cert: "spid-sp.cert"
-
+  remote:
+    - "http://spid-express-app:3000/metadata"
```

Does this need `spid-express-app` to be mapped in the local `hosts` file? Can't we just use `localhost`?

---

This url is called from the `spid-testenv2` container. To work properly the URL must be the follow
```suggestion
    - "http://io-spid-commons:3000/metadata"
```

---

Maybe we can add a comment line which documents this constraint

---

@balanza This is all assumed to brought up with docker-compose, so Docker's networking takes care of providing the hostname. You're right, it needs to be `io-spid-commons`! (we have a different name downstream)